public/jsx/components/submit-button.jsx

```diff
@@ -30,7 +31,7 @@ var SubmitButton = React.createClass({
       <button {...buttonAttrs}
```

Comment:
maybe adjust to ``var { isDisabled, text, ...buttonAttrs }`` at the top of render() so it doesn't stick a text attribute in the button tag? I guess it doesn't matter too much.